src_assets/common/assets/web/config.html

```diff
@@ -591,7 +591,21 @@ <h1 class="my-4">Configuration</h1>
           HEVC is more CPU-intensive to encode, so enabling this may reduce performance when using software encoding.
         </div>
       </div>
-      <!--Encoder -->
+      <!--Capture-->
+      <div class="mb-3" v-if="platform === 'linux'">
+        <label for="capture" class="form-label">Force a Specific Capture Method</label>
+        <select id="capture" class="form-select" v-model="config.capture">
+          <option value>Autodetect</option>
```

Comment:
I'm not an expert in HTML, but should this be `value=""`? They're probably equivalent, so just checking.

Reply:
I copied it from the encoder setting below so I guess it's equivalent. I'll change both though to make it clearer.